newIDE/app/src/UI/SelectField.js

```diff
@@ -34,6 +34,7 @@ type Props = {|
 
   floatingLabelText?: React.Node,
   helperMarkdownText?: ?string,
+  helperText?: React.Node,
```

Comment:
do we have to add this parameter? or can we just re-use the `helperMarkdownText` (even without markdown)

Reply:
Markdown text does not accept anything else than a string as a source so I'm not sure it could be done safely.

Reply:
I think the `SemiControlledTextField` uses a similar prop and has it used sometimes with:
`helperMarkdownText={i18n._(t`As a percent of the game width.`)}`

It would require having i18n imported where the field is used...

I'm not happy with both approaches 😅

Reply:
Yes, I thought of adding this props because it would be the first parameter field to include i18n so not a big fan.

Reply:
switching to new branch https://github.com/4ian/GDevelop/pull/3782 due to conflict hell